meta.yml

```diff
@@ -9,29 +9,17 @@ description: "How federal government domains are meeting best practices on the w
 
 # Data sources. These URLs should have their content updated in-place.
 data:
-  # URL where we download the .gov domain list from.
-  domains_url: "https://github.com/GSA/data/raw/master/dotgov-domains/current-federal.csv"
+  # URL where we download the .ca domain list from.
+  domains_url: "./csv/domains.csv"
```

Comment:
I feel like a bit of pedant for bringing this up, but this is technically no longer a `URL`, it's a `URI`. To make it a URL, we'd have to format it like `file://...`.

For now I think we can leave it, but actually changing the parsing to look for the specific protocol first might be a good refactor in the future.